plugins/check_http.c

```diff
@@ -1724,6 +1724,14 @@ print_help (void)
   printf ("%s\n", _("strings and regular expressions, check connection times, and report on"));
   printf ("%s\n", _("certificate expiration times."));
 
+  printf ("\n");
+  printf ("%s\n", _("ATTENTION!"));
+  printf ("\n");
+  printf ("%s\n", _("This plugin is going to be deprecated. The functionality was reimplemented"));
```

Comment:
Please remove "going to be". This program IS deprecated. It is just still there.

Reply:
I adjusted the text slightly. :)